vizro-core/tests/unit/vizro/actions/test_export_data_action.py

```diff
@@ -237,12 +237,16 @@ def test_multiple_targets_file_format_xlsx(self, callback_context_export_data, g
         )
         assert result["download-dataframe_scatter_chart"]["filename"] == scatter_chart_expected["filename"]
```

Comment:
I've looked into this as well - there is not really a better way to do this check as the results do not seem to be deterministic when using the `to_excel` writer. Decoding and encoding the different base64 strings also do not help.

In this case, I would propose to remove the entire test, so the entire `test_multiple_targets_file_format_xlsx` function. I think it should be fine, because in the end we shouldn't test if the the `dcc.send_data_frame` works properly but what is important to us is that we pass on the **correct dataframe** to the Download component. 

Given that we test for exact content in all other tests using the `to_csv `writer (that does seem to be deterministic), I am fine with removing this test completely.